modules/seo-tools/jetpack-seo.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/**
+ * An SEO expert walks into a bar, bars, pub, public house, Irish pub, drinks, beer, wine, liquor, Grey Goose, Cristal...
+ */
+class Jetpack_SEO {
+	public function __construct() {
+		add_action( 'init', array( $this, 'init' ) );
+	}
+
+	public function init() {
+		if ( apply_filters( 'jetpack_seo_meta_tags', true ) ) {
+			add_action( 'wp_head', array( $this, 'meta_tags' ) );
```

Comment:
From [our earlier conversation](https://github.com/Automattic/jetpack/pull/5307/#discussion_r82974051) (I can't seem to be able to comment there anymore):

> As for non-OG meta tags, I'm wondering if we should plug them there, since all of the OG tags have `<meta property="%s" content="%s" />` format, and SEO tags use `<meta name="%s" content="%s" />`. What do you think?
>
> If the primary reason behind that is plugin compatibility, maybe we could use `jetpack_enable_open_graph` here to check whether other SEO meta tags should be enabled?

That's a good point. We have a filter, `jetpack_open_graph_output`, that we use to change the default markup for specific tags, like here:
https://github.com/Automattic/jetpack/blob/master/class.jetpack-twitter-cards.php#L158-L160

But it might be overkill here. Maybe we should just add `jetpack_enable_open_graph` to allow other plugins and site owners to easily shortcircuit all meta tags, as you suggested. 

@zinigor What do you think about this?